### PR TITLE
fix comments

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
…

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

**Review by wangmingliang-ms — DISMISSED** [review]
LGTM

File: azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/messager/AzureHtmlMessage.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 import static java.util.regex.Pattern.compile;
 
 public class AzureHtmlMessage extends AzureMessage {
-    static final Pattern URL_PATTERN = compile("\\s+https?://(www\\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)");
+    static final Pattern URL_PATTERN = compile("\\s*https?://(www\\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)");
```
Comment (wangmingliang-ms) [inline azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/messager/AzureHtmlMessage.java:23]: need to verify different cases.

Comment (ivywei0125) [inline azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/messager/AzureHtmlMessage.java:23]: consider add test case
- start with space
- start with multi spaces
- start with non-space, such as number, word

File: azure-toolkit-libs/azure-toolkit-applicationinsights-lib/src/main/java/com/microsoft/azure/toolkit/lib/applicationinsights/workspace/LogAnalyticsWorkspaceConfig.java [hunk 2]
```diff
@@ -39,4 +35,18 @@ public static LogAnalyticsWorkspaceConfig createConfig(Subscription subscription
         remoteWorkspace.ifPresent(w -> workspaceConfig.setResourceId(w.getId()));
         return workspaceConfig;
     }
+
```
Comment (ivywei0125) [inline azure-toolkit-libs/azure-toolkit-applicationinsights-lib/src/main/java/com/microsoft/azure/toolkit/lib/applicationinsights/workspace/LogAnalyticsWorkspaceConfig.java]: can revert

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**